pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -405,9 +416,6 @@ spec:
                   deltaSnapshotPeriod:
                     description: DeltaSnapshotPeriod defines the period after which delta snapshots will be taken
                     type: string
-                  fullSnapshotSchedule:
```

Comment:
Why was this removed?

Reply:
It was valid for compaction cronjob which was in last release version of Druid